src/coreclr/jit/lsraarm64.cpp

```diff
@@ -1518,6 +1532,20 @@ int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree, int* pDstCou
                 srcCount++;
             }
         }
+        else if (HWIntrinsicInfo::IsMaskedOperation(intrin.id))
+        {
+            regMaskTP srcCandidates = RBM_NONE;
+            switch (intrin.id)
+            {
+                case NI_Sve_LoadVector:
+                    srcCandidates = RBM_LOWMASK;
```

Comment:
Is `RBM_LOWMASK` true for all variants of `ld1b` or are there some which could operate on higher mask register? I am wondering how we can make this easy for development of other APIs where developer don't have to think about which candidates to set for given intrinsic?